docs/UserGuide.md

```diff
@@ -1,134 +1,233 @@
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
-
-Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
-
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
+> Finds all the tasks with the keywords. <br>
+> Only full words will be matched e.g. meeting will not match meetings
+> Persons matching at least one keyword will be returned (i.e. OR search).
+
+Examples:
+* `find meeting` <br>
+  Returns all tasks with the keyword meeting.
+* `find lunch dinner` <br>
+  Returns any task with lunch or dinner.
+
+<br>
+#### Updating a task: `update`
+Updates the specified task.<br>
+Format: `update INDEX TASK/DATE/TIME/DEADLINE`
+
+> Updates the task at the specific index. The index refers to the index shown in the most recent listing. The index must be a positive integer 1, 2, 3…
+
+Examples:
+* `list`
+* `update 2 project discussion` <br>
+  Updates the 2nd task on the list with taskname project discussion
+* `find dinner`
+* `update 1 6pm-7pm` <br>
+  Updates the 1st task in the results of the `find` command with the new task time
+
+<br>
+#### Deleting a task: `delete`
+Deletes the specified task.<br>
 Format: `delete INDEX`
 
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
-
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
-
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
-
-Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+> Deletes the task at the specific index. The index refers to the index shown in the most recent listing.
+> The index must be a positive integer 1, 2, 3… If the task selected is a recurring task, user will be prompted to delete all instances of the task or only the next instance of the task.
+
+Examples:
+* `list`
+* `delete 2` <br>
+  Deletes the 2nd task on the list
+* `find dinner`
+* `delete 1` <br>
+  Deletes the 1st task in the results of the `find` command
```

Comment:
Can the user delete "dinner" by using a single command if he doesn't know the id?